cmd/repo-init/main.go

```diff
@@ -211,19 +212,28 @@ A test named %s already exists. Please choose a different name.\n`, test.As)
 					break
 				}
 			}
+
+			clusterProfiles := sets.NewString([]string{"gcp", "aws", "azure4"}...)
+
 			test.Profile = api.ClusterProfile(fetchOrDefaultWithPrompt("Which specific cloud provider does the test require, if any? ", string(api.ClusterProfileAWS)))
 			for {
-				if test.Profile.ClusterType() == "" {
+				if !clusterProfiles.Has(string(test.Profile)) {
 					fmt.Printf(`
-No cluster profile named %s exists. Please choose one from: %v.\n`, test.Profile, api.ClusterProfiles())
+Cluster profile %s is not valid. Please choose one from: gcp, aws, azure4.\n`, test.Profile)
 					test.Profile = api.ClusterProfile(fetchOrDefaultWithPrompt("Which specific cloud provider does the test require, if any? ", string(api.ClusterProfileAWS)))
 				} else {
 					break
 				}
 			}
 			test.Command = fetchWithPrompt("What commands in the repository run the test (e.g. \"make test-e2e\")? ")
+
+			if fetchBoolWithPrompt("Does your tests using openshift-client commands? ") {
+				test.Cli = fetchOrDefaultWithPrompt("Which OpenShift client version your tests need to use? ", "latest")
```

Comment:
I don't think we need to expose this - the rest of this tool does not allow the user to create a release that is *not* `latest`